llbuild.xcodeproj/xcshareddata/xcschemes/llbuild-all.xcscheme

```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should we just roll this into the regular llbuild scheme? I'm not sure there is a downside. It might be worth landing that separately, though?